src/index.ts

```diff
@@ -14,22 +14,27 @@ import type {
 	WillShowPrivacyMessage,
 } from './types';
 
+const isServerSide = typeof window === 'undefined';
+
 // Store some bits in the global scope for reuse, in case there's more
 // than one instance of the CMP on the page in different scopes.
-window.guCmpHotFix ||= {};
+if (!isServerSide) {
+	window.guCmpHotFix ||= {};
```

Comment:
Sorry, probably a complete aside, but I found it really difficult to reason around the use of `||=` - maybe just my lack of familiarity though!

Reply:
Oh yes, me too. I read the docs for this but _still_ haven't got a mental model for it

Reply:
I'm thinking about it like this:

```
window.guCmpHotFix = window.guCmpHotFix || {}
```

Reply:
this used to be a very common pattern, similar to @kenoir's example:

`window.guCmpHotFix || (window.guCmpHotFix = {})`

it replaces that iiuc